### DISCUSSION TITLE
Add AWS DataParellism

### DISCUSSION THREAD

**Discussion by nomDeZeus** [Lightning-AI/pytorch-lightning#6560] [DDP / multi-GPU / multi-node] [answered]
Hi!
I currently use AWS SageMaker to train my PL models. I recently found out this link : 
https://aws.amazon.com/fr/blogs/aws/managed-data-parallelism-in-amazon-sagemaker-simplifies-training-on-large-datasets/
SageMaker provides its own implementation of DDP and I think that would be nice to be able to use it with PL :)
I looked into PL code and I think I could add  this feature by extending pytorch_lightning.accelerators.accelerator.Accelerator. Does it seems like a good way to implement it? Are there some general advices/guidance you could give me about this?
If the you are interested in this feature, I can make a PR once I'm done with it.
Thank you!

Rémi

**Comment by carmocca** [comment, marked as answer]
In progress in https://github.com/PyTorchLightning/pytorch-lightning/pull/6271
